x/ccv/provider/proposal_handler_test.go

```diff
@@ -95,6 +103,9 @@ func TestProviderProposalHandler(t *testing.T) {
 			testkeeper.SetupForStoppingConsumerChain(t, ctx, &providerKeeper, mocks)
 
 		case tc.expValidEquivocation:
+			if tc.expValidEquivocation {
```

Comment:
Doesn't seem like there's any test cases where this is true, should prob remove the check or make another tc where it's true

Reply:
There is, check lines above

Reply:
Refactored it tho

Reply:
I don't see where that check is made? Other test cases have `expValidConsumerRemoval` true tho

Reply:
Ah nvm I'm dumb, looks good

Reply:
was rushing my review lol